FirebaseCore/Extension/FIRLibrary.h

```diff
@@ -33,7 +33,7 @@ NS_SWIFT_NAME(Library)
 
 /// Returns one or more Components that will be registered in
 /// FirebaseApp and participate in dependency resolution and injection.
-+ (NSArray<FIRComponent *> *)componentsToRegister;
++ (NSArray<FIRComponent *> *NS_SWIFT_SENDING)componentsToRegister NS_SWIFT_UI_ACTOR;
```

Comment:
@morganchen12 , do you recall why this change?

Reply:
I made this because I wanted to make an entire top-level class (Installations?) `@MainActor`. I can't remember why this had to be made though. You can probably omit it.